reverse_engineering/helpers/oracleHelper.js

```diff
@@ -691,11 +692,13 @@ const getJsonSchema = async (jsonColumns, records) => {
 			return properties;
 		}
 
+		const type = `${schema.type}${subtype.charAt(0).toUpperCase()}${subtype.slice(1)}`
```

Comment:
just advice: if you have non-dynamic property (schema.type is always equals "json"), and it will be static all the time - no need to write it in dynamic way (of course except cases when this better describing this value, like "const seconds = 2*60*60"). 

And also this logic for getting "jsonNumber" string should move to a separate small function named, for example, getJsonTypeName. It will be less confusing

Reply:
Ok, I will do it right now